templates/shared/user/profile_big_avatar.tmpl

```diff
@@ -17,9 +17,11 @@
 		{{if .EnableFeed}}
 			<a href="{{.ContextUser.HomeLink}}.rss"><i class="ui text grey gt-ml-3" data-tooltip-content="{{.locale.Tr "rss_feed"}}">{{svg "octicon-rss" 18}}</i></a>
 		{{end}}
-		<div class="gt-mt-3">
-			<a class="muted" href="{{.ContextUser.HomeLink}}?tab=followers">{{svg "octicon-person" 18 "gt-mr-2"}}{{.NumFollowers}} {{.locale.Tr "user.followers"}}</a> · <a class="muted" href="{{.ContextUser.HomeLink}}?tab=following">{{.NumFollowing}} {{.locale.Tr "user.following"}}</a>
-		</div>
+		{{if or (or .NumFollowers .NumFollowing) (not .RunModeIsProd)}}
```

Comment:
```suggestion
		{{if or .NumFollowers .NumFollowing)}}
```

Not a fan of mode-specific display here as well.

Reply:
I am definitely getting mixed signals over here, but I'll remove it.